.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Tell us about a problem you are experiencing
+
+---
+
+**What steps did you take and what happened:**
+[A clear and concise description of what the bug is, and what commands you ran.)
+
+
+**What did you expect to happen:**
+
+
+**The output of the following commands will help us better understand what's going on**:
+(Pasting long output into a [GitHub gist](https://gist.github.com) or other pastebin is fine.)
+
+* `kubectl logs <jiva controller pod name> -n openebs`
```

Comment:
mayactl volume info command also will be good to fetch here.

Reply:
And the status of the pods. 

the users may hesitate to provide the logs.

Reply:
@kmova ok, i'm putting this as optional.